exporter/collector/metrics.go

```diff
@@ -949,40 +983,54 @@ func (m *metricMapper) labelDescriptors(
 	return result
 }
 
-// Returns (sum, count) metric names for a summary metric.
-func summaryMetricNames(name string) (string, string) {
-	sumName := fmt.Sprintf("%s%s", name, SummarySumSuffix)
-	countName := fmt.Sprintf("%s%s", name, SummaryCountPrefix)
-	return sumName, countName
+// Returns (sum, count, quantile) metric names for a summary metric.
+func (m *metricMapper) summaryMetricNames(pm pdata.Metric) (string, string, string, error) {
```

Comment:
nit: `summaryMetricTypes` is a better name.

Reply:
done